src/display.rs

```diff
@@ -143,80 +161,67 @@ fn inner_display_tree(
     flags: &Flags,
     colors: &Colors,
     icons: &Icons,
-    depth: usize,
-    prefix: &str,
-) -> String {
-    let mut output = String::new();
+    tree_depth_prefix: (usize, &str),
```

Comment:
IMO, I think it would be cleaner to keep `depth` and `prefix` as two separate values itself.

Reply:
yes, I change this only because of the `too_many_arguments` warning, sigh...

and I can not figure out a better way to reduce the number of args, is there a better solution in your sight?

```
warning: this function has too many arguments (8/7), #[warn(too_many_arguments)] on by default
```